web/src/main/java/com/epam/rft/atsy/web/configuration/ExceptionMapperConfiguration.java

```diff
@@ -12,47 +12,68 @@
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordNewMatchValidationException;
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordOldMatchValidationException;
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordUniqueValidationException;
-import com.epam.rft.atsy.web.mapper.Rule;
+import com.epam.rft.atsy.web.mapper.ExceptionMessagePair;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * This configuration class is responsible for the bean that stores all the validation exceptions
+ * along with their corresponding messages. The bean is filled with data here, so if some changes
+ * happen among the validation exceptions or the exception messages, this class must be adjusted
+ * according to the changes.
+ */
 @Configuration
 public class ExceptionMapperConfiguration {
 
+  /**
```

Comment:
The javadocs are without line breaks after the summary sentence because of the current code formatting conventions that we applied to this project.